tests/config/run_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Tends to be a bit more portable